run_tests.sh

```diff
@@ -17,7 +17,7 @@
 set -e
 
 # Array of top-level folders with Python code.
-PYTHON_FOLDERS="util/ scripts/ import-automation/"
+PYTHON_FOLDERS="util/ scripts/ import-automation/executor"
```

Comment:
I'm not sure how the existing tests were even discovering the tests because the base directory was import-automation. Changing it to import-automation/executor to test just executor python tests (can't get dashboard API and UI to run right away, will leave that for future)

Reply:
Thanks for the find.  Could you have a TODO / GH issue  for that?

Reply:
Filed https://github.com/datacommonsorg/data/issues/836